cpp/benchmarks/io/parquet/parquet_reader_input.cpp

```diff
@@ -272,6 +333,24 @@ NVBENCH_BENCH(BM_parquet_read_io_small_mixed)
   .add_int64_axis("run_length", {1, 32})
   .add_int64_axis("num_string_cols", {1, 2, 3});
 
+using d_type_list_wide_table = nvbench::enum_type_list<data_type::DECIMAL, data_type::STRING>;
+NVBENCH_BENCH_TYPES(BM_parquet_read_wide_tables, NVBENCH_TYPE_AXES(d_type_list_wide_table))
+  .set_name("parquet_read_wide_tables")
+  .set_min_samples(4)
+  .set_type_axes_names({"data_type"})
+  .add_int64_axis("data_size_mb", {512, 2048, 4095})
+  .add_int64_axis("num_cols", {64, 512, 1024})
+  .add_int64_axis("cardinality", {0, 1000})
+  .add_int64_axis("run_length", {1, 32});
+
+NVBENCH_BENCH(BM_parquet_read_wide_tables_mixed)
+  .set_name("parquet_read_wide_tables_mixed")
+  .set_min_samples(4)
+  .add_int64_axis("data_size_mb", {512, 2048, 4095})
+  .add_int64_axis("num_cols", {64, 512, 1024})
```

Comment:
We might want to have geometric progressions for the parameters. e.g. 1024, 2048, 4096 for table size and then 256, 512, 1024 for column count. This way we can answer questions like "for 
a fixed number of rows (i.e. column size in mb, i.e. `data_size_mb/num_cols`), how much penalty (or benefit?) do we have for wider tables". If we don't increment the two parameters evenly, we don't have (nearly as many) multiple cases with equal `data_size_mb/num_cols`.